src/scss/win/_type.scss

```diff
@@ -1,139 +1,127 @@
-    @include win-type-header();
-}
-h2, .h2, .win-type-subheader, .text-subheader {
-    @include win-type-subheader();
-}
-h3, .h3, .win-type-title, .text-title {
-    @include win-type-title();
-}
-h4, .h4, .win-type-subtitle, .text-subtitle {
-    @include win-type-subtitle();
-}
-h5, .h5, .win-type-base, .text-base {
-    @include win-type-base();
-}
-p, .win-type-body, .text-body {
-    @include win-type-body();
-}
-h6, .h6, .win-type-caption, .text-caption {
-    @include win-type-caption();
-}
-.win-type-caption-alt, .text-caption-alt {
-   @include win-type-caption-alt();
-}
-.win-type-subcaption, .text-subcaption {
-    @include win-type-subcaption();
-}
-
+﻿// Icon font
+@font-face {
+  font-family: "winjs-symbols";
+  src: url("#{$icon-font-path}/winjs-symbols.eot");
+  src: url("#{$icon-font-path}/winjs-symbols.eot#iefix") format("embedded-opentype"),
+       url("#{$icon-font-path}/winjs-symbols.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Selawik";
+  src: url("../fonts/selawk.eot");
+  src: url("../fonts/selawk.eot#iefix") format("embedded-opentype"),
+       url("../fonts/selawk.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Selawik Bold";
+  src: url("../fonts/selawkb.eot");
+  src: url("../fonts/selawkb.eot#iefix") format("embedded-opentype"),
+       url("../fonts/selawkb.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Selawik Light";
+  src: url("../fonts/selawkl.eot");
+  src: url("../fonts/selawkl.eot#iefix") format("embedded-opentype"),
+       url("../fonts/selawkl.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Selawik Semibold";
+  src: url("../fonts/selawksb.eot");
+  src: url("../fonts/selawksb.eot#iefix") format("embedded-opentype"),
+       url("../fonts/selawksb.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Selawik Semilight";
+  src: url("../fonts/selawksl.eot");
+  src: url("../fonts/selawksl.eot#iefix") format("embedded-opentype"),
+       url("../fonts/selawksl.ttf") format("truetype");
+}
+
+@mixin responsive-type($font) {
+    
+    font-family:        map-deep-get($win-fonts, $font, "font-family");
+    font-size:          map-deep-get($win-fonts, $font, "size-xs");
+    font-weight:        map-deep-get($win-fonts, $font, "font-weight-xs");
+    line-height:        (map-deep-get($win-fonts, $font, "line-height-xs") / map-deep-get($win-fonts, $font, "size-xs"));
+
+    @media (min-width: $screen-md-min) {
+        font-size:      map-deep-get($win-fonts, $font, "size-md");
+        line-height:    (map-deep-get($win-fonts, $font, "line-height-md") / map-deep-get($win-fonts, $font, "size-md"));
+    }
+    @media (min-width: $screen-xl-min) {
+        font-size:      map-deep-get($win-fonts, $font, "size-xl");
+        line-height:    (map-deep-get($win-fonts, $font, "line-height-xl") / map-deep-get($win-fonts, $font, "size-xl"));
+    }
+
+}
+
+// Responsive Typography mixins
+@mixin win-type-headline {

```

Comment:
as above, without adding h1,.h1, etc to these classes, seems like it'll be more of a "port" than a "theme" to take a bootstrap site and consume Winstrap.